maven-projects/info/src/main/java/org/apache/graphar/util/PathUtil.java

```diff
@@ -19,9 +19,16 @@
 
 package org.apache.graphar.util;
 
+import java.nio.file.Path;
+
 public class PathUtil {
+
+    public static String resolvePath(String basePath, String subPath) {
+        return Path.of(basePath).resolve(subPath).toString();
+    }
+
     public static String pathToDirectory(String path) {
-        if (path.startsWith("s3://")) {
+        if (path.startsWith("s3://") || path.startsWith("oss://")) {
```

Comment:
We can get the protocol name simply by `URI.getScheme()` as well, as we can get path by `URI.getPath()`